### ISSUE TITLE
Cipher suite configuration per SNI?

### ISSUE THREAD

**Issue by mtorluemke** [apache/trafficserver#7174]
Does it make sense to be able to have cipher suites per SNI? (Today, you can specify TLS versions, but not cipher suites.)

**Comment by shinrich** [comment]
Yes, that does make sense.  Just never got around to it.  A pressing need has not come up yet for us, bit it should not be too hard to add.  Need to ensure that the appropriate openssl API exists for that.  I'll go do some research on that.

**Comment by bryancall** [comment]
This is now supported. PR https://github.com/apache/trafficserver/pull/12110 added two per Server Name Indication (SNI) keys to sni.yaml: `server_cipher_suite` (overrides the global `proxy.config.ssl.server.cipher_suite` for pre TLSv1.3) and `server_TLSv1_3_cipher_suites` (overrides `proxy.config.ssl.server.TLSv1_3.cipher_suites`). These let you configure inbound cipher suites on a per SNI basis, which is what this issue requested. The change first shipped in release 10.1.0. Closing as implemented.
